### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by satejwagle**
Added Files

### CODE REVIEW

File: scrape3.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+       l= l+1
+            
+df.to_csv('/Users/satejwagle/Seva Setu/sample.csv')
+results[1]
+
+session = requests.session()
+rj = session.get("https://india.gov.in/my-government/schemes")
+token = find_between( str(rj.content), "theme_token\":\"", "\"" )
+
+50a1555819ee6ac393015ac41fe81d26
```
Comment (shashank-srikant) [inline scrape3.py:71]: Are y ou sure this is compilable code? Can you please check?

**Review by shashank-srikant — COMMENTED** [review]
Can you also please add the module to scrape the previous webpage we attempted? Even though it is temporary and is yet to be fully fleshed out, will be good to upload to repo and keep a track of the changes you're making on it.

Also, let's have one file each for every web page that we attempt to scrape.

Thanks.
Shashank

File: scrape3.py (hunk 1 above)
Comment (satejwagle) [reply scrape3.py:71]: No this isn't compilable code. Need to make some changes to make it work.

Comment (shashank-srikant) [reply scrape3.py:71]: I meant by looking at the last line with the string of random characters.

**Pull request merged**